### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by Olexandr88**
Update contributing

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15278/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15278/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15278/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15278/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15278/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15278/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15278/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15278/merge#BCU1XR#0

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15278/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15278/merge/testResults/webgpuplaywright/index.html

File: contributing.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The first golden rule is a really important one because we want our users to tru
 
 ### Performance
 
-Babylon.js is a 3D rendering engine. So every piece of code has to be scrutinized to look for potential bottlenecks or slowdowns. Ultimately the goal is to render more with less resources.
+Babylon.js is one of the most popular 3D game engines used by developers. As with any other 3D library it provides built-in functions to help you implement common 3D functionality more quickly.
```
Comment (deltakosh) [inline contributing.md:17]: can you explain why you want that change?

Comment (Olexandr88) [reply contributing.md:17]: Hello. In my opinion, these changes better describe the principle and function of the project

**Comment by RaananW** [comment]
Closing this, as this is not needed ATM. thanks!

**Comment by Olexandr88** [comment]
Well that's your right

**Pull request closed without merging**